packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -773,8 +773,8 @@
 		"noMatchingTagsExist": "No matching tags exist",
 		"noTagsExist": "No tags exist",
 		"showError": {
-			"message": "A problem occurred when trying to create the {name} tag",
-			"title": "New tag was not created"
+			"message": "A problem occurred when trying to create the '{name}' tag",
+			"title": "Couldn't create tag"
```

Comment:
For consistency, others above do not use the contraction: `couldn't` -> `could not`

Same for other instances of `couldn't` below.